app/components/ember-survey.js

```diff
@@ -5,7 +5,6 @@ import {computed } from '@ember/object';
 import { pluralize } from 'ember-inflector';
 
 export default Component.extend({
-  tagName: '',
```

Comment:
@BradenLawrence I plan to Glimmerize the components at a later time. Can we revert this change?

Reply:
This was a fix for an error in one of the visual regression tests:
```
not ok 1 Chrome 85.0 - [7624 ms] - Acceptance | visual regression: visiting visual regressions with Percy
    ---
        actual: >
            [object Object]
        stack: >
            Error: Assertion Failed: calling set on destroyed object: <ember-website@component:high-charts::ember566>.highcharts = [object Object]
                at assert (http://localhost:7357/assets/vendor.js:37175:15)
                at set (http://localhost:7357/assets/vendor.js:16141:49)
                at Class.set (http://localhost:7357/assets/vendor.js:28978:29)
                at http://localhost:7357/assets/ember-website.js:723:14
        message: >
            Assertion Failed: calling set on destroyed object: <ember-website@component:high-charts::ember566>.highcharts = [object Object]
        negative: >
            false
        browser log: |
            LOG: [percy] Error fetching DOM library, disabling: TypeError: Failed to fetch
    ...
```

I paired with @jenweber on this, and she found that it was because using a modifier on an element using `tagName: ''` would fail because there would no longer be an element to modify ([reference](https://github.com/emberjs/ember-render-modifiers/issues/15)) Is there a different way we could handle this?

Reply:
Hmm, I'm a bit confused. I'm looking at the codebase and currently don't see how `<EmberSurvey>` accepts a modifier.

The test's error message seems to indicate that `<HighCharts>` component may be causing an issue (quite possible). Usually, the message `calling set on destroyed object` tends to indicate that a component tried to complete doing something _after_ we visit another page that doesn't have the component.

Reply:
Gotcha, since this seems to be an issue with `<HighCharts>`, I think it would make sense to revert this change and handle the failing test in a different PR. I'll do that and resubmit.